### DISCUSSION TITLE
update logo?

### DISCUSSION THREAD

**Discussion by raybellwaves** [anitagraser/movingpandas#241] [General]
Most PyData projects have a logo to the left and their name to the right e.g. https://geopandas.org/en/stable/_static/geopandas_logo_web.svg

Could keep the logo the same and have a version with "MovingPandas" to the right of it.

If there is any info on how the logo is generated I would be happy to create it

**Comment by anitagraser** [comment]
We can certainly create a new (additional) version of the logo. It's available as SVG if you want to give it a go: 

![logo](https://user-images.githubusercontent.com/590385/191930516-09e82170-4cac-4d79-acce-bf12e75b39f2.svg)

It will be good to have both versions so we can use the most appropriate one for any given occasion.

**Comment by anitagraser** [comment]
![logo-wide](https://user-images.githubusercontent.com/590385/192142572-9572ccb6-47f6-45b7-af8e-a633f5b299b0.png)

I've added all SVGS to the gh-pages branch:  https://github.com/anitagraser/movingpandas/commit/119c58f8341c62899c61827f6c2fde81775aa072
